applications/app/controllers/CrosswordsController.scala

```diff
@@ -350,7 +354,25 @@ class CrosswordEditionsController(
     "crosswords/series/quiptic",
   ).mkString("|")
 
-  private def parseCrosswords(response: SearchResponse): EditionsCrosswordRenderingDataModel =
-    EditionsCrosswordRenderingDataModel(response.results.flatMap(_.crossword))
+  private def parseCrosswords(response: SearchResponse): Seq[EditionsCrosswordRenderingDataModel] = {
+    response.results.flatMap(_.crossword).map { crossword =>
+      EditionsCrosswordRenderingDataModel(
+        id = s"crosswords/${crossword.`type`.name}/${crossword.number}",
+        number = crossword.number,
+        name = crossword.name,
+        creator = crossword.creator,
+        date = crossword.date.toJoda.getMillis,
+        webPublicationDate = crossword.date.toJoda.getMillis,
+        entries =
+          crossword.entries.map(entry => EditionsCrosswordEntry.fromCrosswordEntry(entry, crossword.solutionAvailable)),
+        solutionAvailable = crossword.solutionAvailable,
+        dateSolutionAvailable = crossword.dateSolutionAvailable.map(_.toJoda.getMillis).getOrElse(0L),
+        dimensions = crossword.dimensions,
+        crosswordType = CamelCase.toHyphenated(crossword.`type`.name),
+        pdf = crossword.pdf,
+        instructions = crossword.instructions,
+      )
```

Comment:
You may want to put this in an `apply` method on the `EditionsCrosswordRenderingDataModel` companion object as before? That way the model transformations happen all together, and close to the model, rather than some in the model file and some in the controller?